internal/api.go

```diff
@@ -579,7 +579,9 @@ func logf(c *context, level int64, format string, args ...interface{}) {
 		Level:         &level,
 		Message:       &s,
 	})
-	log.Print(logLevelName[level] + ": " + s)
+	if !IsSecondGen() {
```

Comment:
Can you add a comment like "Only duplicate log to stderr if not running on App Engine second generation"